resources/assets/js/plugins/ckeditor.js

```diff
@@ -0,0 +1,18 @@
+window.CKEDITOR_BASEPATH = '/plugins/ckeditor/';
+
+var selector = $('.js-ckeditor'),
+    ckeditor_script = CKEDITOR_BASEPATH + 'ckeditor.js',
+    ckeditor_adapter = CKEDITOR_BASEPATH + 'adapters/jquery.js';
+
+if (selector.length > 0) {
+    $.getScript(ckeditor_script, function(data) {
+
+        $.getScript(ckeditor_adapter, function(data) {
```

Comment:
Identifier 'ckeditor_adapter' is not in camel case.<br>'data' is defined but never used.